source/includes/administration/_organizations.md

```diff
@@ -467,9 +533,88 @@ Optional | &nbsp;
 ---- | ----
 `autoCreationEnabled`<br/>*boolean* | A boolean specifying whether to enable automatic end-user account creation upon successful OIDC login.
 `verifiedDomains`<br/>*Array[[verified domains](#administration-get-verified-domains)]*| A list of objects containing the ids of verified domains (with VERIFIED status) for which successful matching OIDC logins will create new users.
+`passwordPolicy`<br/>*object*  | The password policy that will be assigned to the organization. 
+`passwordPolicy.constraints`<br/>*Array[Object]* | List of password policy constraints objects with the following fields.
+`passwordPolicy.constraints.name`<br/>*string* | A string that represents the constraint name.
+`passwordPolicy.constraints.value`<br/>*int* | An integer that represents the minimum value for the constraint.
+`passwordPolicy.constraints.isMandatory`<br/>*boolean* | A boolean flag to indicate if the constraint is mandatory or not.
 
 Returns an HTTP status code 200, with an empty response body.
 
+- If `defaultRole` is not passed in the request:
+   - The system will assign `Guest`as a default role to the organization.
+- If `passwordPolicy` is passed in the request:
+   - The system will create a new or update the existing password policy for the organization.
+- If `autoCreationEnabled` and/or `verifiedDomains` is passed in the request:
+   - The system will create a new or update the existing security settings for the organization.
+
```

Comment:
Looks good.
We can even put this info in a notice section something like -

![image](https://user-images.githubusercontent.com/5191515/111373028-d0f3d800-8671-11eb-8148-e4f7c0a8b8b7.png)

by putting it all inside `<aside class="notice"></aside>` tag.

Reply:
I tried using the `<aside class="notice"></aside>` tag, but the other formatting options go for a toss in that case. IMO, the previous one with proper formatting also looks neet.

![image](https://user-images.githubusercontent.com/20757399/111376139-d6efb600-867d-11eb-8d55-6e220ce56d8a.png)

Reply:
Thanks for checking.